newIDE/app/src/UI/User/UserChip.js

```diff
@@ -28,12 +28,7 @@ type Props = {|
 
 const UserChip = ({ onOpenProfile }: Props) => {
   const authenticatedUser = React.useContext(AuthenticatedUserContext);
-  const {
-    profile,
-    onOpenLoginDialog,
-    onOpenCreateAccountDialog,
-    loginState,
-  } = authenticatedUser;
+  const { profile, onOpenCreateAccountDialog, loginState } = authenticatedUser;
   // TODO: Remove the badge on the user chip and handle badge notifications
   // with user notifications.
   const displayNotificationBadge = hasPendingBadgeNotifications(
```

Comment:
maybe take the opportunity to remove the notification badge? I remember we discussed this wasn't that useful as not really understood

Reply:
Good idea, doing this. If you get a new achievement and we want you to know, we should send a "real" notification now instead :)

Reply:
Done